test/system/160-volumes.bats

```diff
@@ -619,4 +619,45 @@ EOF
     run_podman volume rm $volume_name --force
 }
 
+@test "podman volumes with XFS quotas" {
+    skip_if_rootless "Quotas are only possible with root"
+    skip_if_remote "Requires --root flag, not possible w/ remote"
+
+    # Minimum XFS filesystem size is 300mb
+    loop=$PODMAN_TMPDIR/loopdevice
+    dd if=/dev/zero of=$loop bs=1M count=300
+    loop_dev=$(losetup -f)
+    losetup $loop_dev $loop
+    mkfs.xfs $loop_dev
```

Comment:
```
    # Minimum XFS filesystem size is 300mb
    loop=$PODMAN_TMPDIR/disk.img
    fallocate -l 300m  ${loop}
    run -0 losetup -f --show $loop
    loop_dev="$output"
    mkfs.xfs $loop_dev
```

This should be a bit more race safe I think and we may not have to copy 300mb of zeros.

And I still think you need to move this to 180 and then add proper teardown logic to unmount the fs and loop device.

Reply:
Would you object to me adding a 161-volume-quotas.bats and throwing this test in there along with the cleanup? I hesitate to throw things in unrelated files